manual/pyscf_for_afqmc.tex

```diff
@@ -0,0 +1,132 @@
+\item Run a standard pyscf calculation, e.g. a HF or DFT calculation. Make sure you preserve the chkfile and make sure you store the core hamiltonian on the chkfile. An example of how to do this for a single k-point calculation is found below.
+\begin{lstlisting}[caption=The following is an example PYSCF input file for single k-point calculations.]
+import numpy
+import h5py
+from mpi4py import MPI
+from pyscf.pbc import gto, scf, dft, cc
+
+cell = gto.Cell()
+cell.a = '''
+3.5668 0 0
+0 3.5668 0
+0 0 3.5668'''
+cell.atom = '''C 0. 0. 0. 
+C 0.8917 0.8917 0.8917
+C 1.7834 1.7834 0. 
+C 2.6751 2.6751 0.8917
+C 1.7834 0. 1.7834
+C 2.6751 0.8917 2.6751
+C 0. 1.7834 1.7834
+C 0.8917 2.6751 2.6751'''
+cell.basis = 'gth-szv'
+cell.pseudo = 'gth-pade'
+cell.gs = [10]*3 # 10 grids on postive x direction, => 21^3 grids in total
+cell.verbose = 4
+cell.build()
+
+kpt = cell.get_abs_kpts([0.25, 0.25, 0.25])  
+
+mf = scf.RHF(cell,kpt,exxdiv=0)
+mf.chkfile = "scf.dump"                         # store checkpoint file in scf.dump
+ehf = mf.kernel()
+print("HF energy (per unit cell) = %.17g" % ehf)
+
+hcore = mf.get_hcore(kpt=kpt)            # obtain and store core hamiltonian
+with h5py.File(mf.chkfile) as fh5:
+  fh5['scf/hcore'] = hcore
+\end{lstlisting}
+
+\item {For a calculation with k-points:
+Run a standard pyscf calculation, e.g. a HF or DFT calculation. Make sure you preserve the chkfile and make sure you store the core hamiltonian on the chkfile. An example of how to do this for a single k-point calculation is found below.}
+
+\begin{lstlisting}[caption=The following is an example PYSCF input file for calculations with k-points.]
+import numpy
+import h5py
+from mpi4py import MPI
+from pyscf.pbc import gto, scf, dft, cc
+
+cell = gto.Cell()
+cell.a = '''
+3.5668 0 0
+0 3.5668 0
+0 0 3.5668'''
+cell.atom = '''C 0. 0. 0. 
+C 0.8917 0.8917 0.8917
+C 1.7834 1.7834 0. 
+C 2.6751 2.6751 0.8917
+C 1.7834 0. 1.7834
+C 2.6751 0.8917 2.6751
+C 0. 1.7834 1.7834
+C 0.8917 2.6751 2.6751'''
+cell.basis = 'gth-szv'
+cell.pseudo = 'gth-pade'
+cell.gs = [10]*3 # 10 grids on postive x direction, => 21^3 grids in total
+cell.verbose = 4
+cell.build()
+
+nk = [2,2,2]
+kpts = cell.make_kpts(nk) 
+
+mf = scf.KRHF(cell,kpts,exxdiv=0)
+mf.chkfile = "scf.dump"                         # store checkpoint file in scf.dump
+ehf = mf.kernel()
+print("HF energy (per unit cell) = %.17g" % ehf)
+
+hcore = mf.get_hcore(kpts=kpts)          # obtain and store core hamiltonian
+fock = (hcore + mf.get_veff(kpts=kpts))  # store fock matrix (required with orthoAO=True)
+with h5py.File(mf.chkfile) as fh5:
+  fh5['scf/hcore'] = hcore
+  fh5['scf/fock'] = fock
+\end{lstlisting}
+\end{itemize}
+
+Once the checkpoint file has been created, it is now possible to generate the integral file. The recommended approach is:
+
+\begin{lstlisting}[caption=The following is an example input file for calculating the integrals.]
+from mpi4py import MPI
+from qmctools import integrals_from_chkfile
```

Comment:
Where does 'qmctools' come from?

Reply:
Still need to upload it and describe it properly.

Reply:
Removed this until further notice.